examples/mobilenet2_example.py

```diff
@@ -0,0 +1,98 @@
+import torch
+from torch import nn
+from torchvision import transforms, datasets
+from torch.utils.data import DataLoader
+
+from tqdm import tqdm
+
+from bitnet.nn.bitlinear import BitLinear
+from bitnet.nn.bitconv2d import BitConv2d
+from bitnet.models.resnet import resnet18
+from bitnet.models.mobilenet2 import create_mobilenet2, create_bit_mobilenet2
```

Comment:
Not a fan of the `create` functions.
I wanted to keep the interface as similar as possible to the "standard" ones.